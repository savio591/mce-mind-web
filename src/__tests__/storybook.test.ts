import initStoryshots from '@storybook/addon-storyshots';

describe('Components: storybook tests', () => {
  initStoryshots({
    configPath: './.storybook',
    framework: 'react',
  });
});
