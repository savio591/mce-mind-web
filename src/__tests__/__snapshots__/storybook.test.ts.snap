// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components: storybook tests Storyshots Components/Button Default 1`] = `
<button
  data-size="default"
  data-style="default"
  data-type="default"
  disabled={false}
  type="button"
>
  Entrar
</button>
`;

exports[`Components: storybook tests Storyshots Components/Button Icon 1`] = `
<button
  data-size="default"
  data-style="default"
  data-type="icon"
  disabled={false}
  type="button"
>
  <svg
    fill="currentColor"
    height={35}
    stroke="currentColor"
    strokeWidth="0"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 512 512"
    width={35}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M80 368H16a16 16 0 0 0-16 16v64a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16v-64a16 16 0 0 0-16-16zm0-320H16A16 16 0 0 0 0 64v64a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16V64a16 16 0 0 0-16-16zm0 160H16a16 16 0 0 0-16 16v64a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16v-64a16 16 0 0 0-16-16zm416 176H176a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16zm0-320H176a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16V80a16 16 0 0 0-16-16zm0 160H176a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16z"
    />
  </svg>
</button>
`;

exports[`Components: storybook tests Storyshots Components/Button Loading 1`] = `
<button
  data-size="default"
  data-style="default"
  data-type="default"
  disabled={true}
  type="button"
>
  Carregando...
</button>
`;

exports[`Components: storybook tests Storyshots Components/Button Outline 1`] = `
<button
  data-size="small"
  data-style="outline"
  data-type="default"
  disabled={false}
  type="button"
>
  Não disponível
</button>
`;

exports[`Components: storybook tests Storyshots Components/Button Small 1`] = `
<button
  data-size="small"
  data-style="default"
  data-type="default"
  disabled={false}
  type="button"
>
  Disponível
</button>
`;

exports[`Components: storybook tests Storyshots Components/Task Archived 1`] = `
<div
  data-state="TASK_ARCHIVED"
>
  <label
    className="checkbox"
    htmlFor="input_1"
  >
    <input
      defaultChecked={true}
      disabled={true}
      id="input_1"
      name="checked"
      type="checkbox"
    />
    <span
      aria-hidden={true}
      onClick={[Function]}
    />
  </label>
  <div>
    <input
      placeholder="Input title"
      readOnly={true}
      type="text"
      value="Test Task"
    />
  </div>
  <div
    aria-hidden={true}
    onClick={[Function]}
  />
</div>
`;

exports[`Components: storybook tests Storyshots Components/Task Default 1`] = `
<div
  data-state="TASK_INBOX"
>
  <label
    className="checkbox"
    htmlFor="input_1"
  >
    <input
      defaultChecked={false}
      disabled={true}
      id="input_1"
      name="checked"
      type="checkbox"
    />
    <span
      aria-hidden={true}
      onClick={[Function]}
    />
  </label>
  <div>
    <input
      placeholder="Input title"
      readOnly={true}
      type="text"
      value="Test Task"
    />
  </div>
  <div
    aria-hidden={true}
    onClick={[Function]}
  >
    <a
      aria-hidden={true}
      onClick={[Function]}
    >
      <span
        className="icon-star"
      />
    </a>
  </div>
</div>
`;

exports[`Components: storybook tests Storyshots Components/Task Pinned 1`] = `
<div
  data-state="TASK_PINNED"
>
  <label
    className="checkbox"
    htmlFor="input_1"
  >
    <input
      defaultChecked={false}
      disabled={true}
      id="input_1"
      name="checked"
      type="checkbox"
    />
    <span
      aria-hidden={true}
      onClick={[Function]}
    />
  </label>
  <div>
    <input
      placeholder="Input title"
      readOnly={true}
      type="text"
      value="Test Task"
    />
  </div>
  <div
    aria-hidden={true}
    onClick={[Function]}
  >
    <a
      aria-hidden={true}
      onClick={[Function]}
    >
      <span
        className="icon-star"
      />
    </a>
  </div>
</div>
`;

exports[`storybook tests Storyshots Components/Button Default 1`] = `
<button
  data-size="default"
  data-style="default"
  data-type="default"
  disabled={false}
  type="button"
>
  Entrar
</button>
`;

exports[`storybook tests Storyshots Components/Button Icon 1`] = `
<button
  data-size="default"
  data-style="default"
  data-type="icon"
  disabled={false}
  type="button"
>
  <svg
    fill="currentColor"
    height={35}
    stroke="currentColor"
    strokeWidth="0"
    style={
      Object {
        "color": undefined,
      }
    }
    viewBox="0 0 512 512"
    width={35}
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M80 368H16a16 16 0 0 0-16 16v64a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16v-64a16 16 0 0 0-16-16zm0-320H16A16 16 0 0 0 0 64v64a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16V64a16 16 0 0 0-16-16zm0 160H16a16 16 0 0 0-16 16v64a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16v-64a16 16 0 0 0-16-16zm416 176H176a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16zm0-320H176a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16V80a16 16 0 0 0-16-16zm0 160H176a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16z"
    />
  </svg>
</button>
`;

exports[`storybook tests Storyshots Components/Button Loading 1`] = `
<button
  data-size="default"
  data-style="default"
  data-type="default"
  disabled={true}
  type="button"
>
  Carregando...
</button>
`;

exports[`storybook tests Storyshots Components/Button Outline 1`] = `
<button
  data-size="small"
  data-style="outline"
  data-type="default"
  disabled={false}
  type="button"
>
  Não disponível
</button>
`;

exports[`storybook tests Storyshots Components/Button Small 1`] = `
<button
  data-size="small"
  data-style="default"
  data-type="default"
  disabled={false}
  type="button"
>
  Disponível
</button>
`;

exports[`storybook tests Storyshots Components/Task Archived 1`] = `
<div
  data-state="TASK_ARCHIVED"
>
  <label
    className="checkbox"
    htmlFor="input_1"
  >
    <input
      defaultChecked={true}
      disabled={true}
      id="input_1"
      name="checked"
      type="checkbox"
    />
    <span
      aria-hidden={true}
      onClick={[Function]}
    />
  </label>
  <div>
    <input
      placeholder="Input title"
      readOnly={true}
      type="text"
      value="Test Task"
    />
  </div>
  <div
    aria-hidden={true}
    onClick={[Function]}
  />
</div>
`;

exports[`storybook tests Storyshots Components/Task Default 1`] = `
<div
  data-state="TASK_INBOX"
>
  <label
    className="checkbox"
    htmlFor="input_1"
  >
    <input
      defaultChecked={false}
      disabled={true}
      id="input_1"
      name="checked"
      type="checkbox"
    />
    <span
      aria-hidden={true}
      onClick={[Function]}
    />
  </label>
  <div>
    <input
      placeholder="Input title"
      readOnly={true}
      type="text"
      value="Test Task"
    />
  </div>
  <div
    aria-hidden={true}
    onClick={[Function]}
  >
    <a
      aria-hidden={true}
      onClick={[Function]}
    >
      <span
        className="icon-star"
      />
    </a>
  </div>
</div>
`;

exports[`storybook tests Storyshots Components/Task Pinned 1`] = `
<div
  data-state="TASK_PINNED"
>
  <label
    className="checkbox"
    htmlFor="input_1"
  >
    <input
      defaultChecked={false}
      disabled={true}
      id="input_1"
      name="checked"
      type="checkbox"
    />
    <span
      aria-hidden={true}
      onClick={[Function]}
    />
  </label>
  <div>
    <input
      placeholder="Input title"
      readOnly={true}
      type="text"
      value="Test Task"
    />
  </div>
  <div
    aria-hidden={true}
    onClick={[Function]}
  >
    <a
      aria-hidden={true}
      onClick={[Function]}
    >
      <span
        className="icon-star"
      />
    </a>
  </div>
</div>
`;
